Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PhysicalPressButton.prefab

```diff
@@ -0,0 +1,412 @@
+%YAML 1.1
```

Comment:
Interesting to see that you're adding a second pressable button to MRTK SDK. It might be good to think about the customer perspective here.

Putting myself into shoes of external customer, I would be confused about whether I should use PressableButtonPlated.prefab (old prefab) or PhysicalPressButton.prefab.

Assuming that our goal is to make implementing UX as smooth as possible, I wonder if adding another option would actually create confusion / stress for the customer, because they may not know what to do.

A few options I see here:
- Agree that we have two types of buttons and in the prefab naming make it clear what each one is for (e.g. InteractablePhysicalPressbutton.prefab).
- Pull back the change from SDK, and just put this in examples, so that there is just one kind of button in SDK.

Reply:
Agreed, if we figure out the above discussions about what functionality needs be here, that will effect which samples we create and what they will be named.

Reply:
removed pressable button examples and prefabs